plat/kvm/Config.uk

```diff
@@ -126,35 +118,66 @@ config KVM_DEBUG_VGA_CONSOLE
         help
           Choose VGA console for the debug printing
 
+config KVM_KERNEL_SERIAL_CONSOLE
+        bool "Serial console for the kernel prints"
+        default y
+        depends on (ARCH_X86_64 || ARCH_ARM_64)
```

Comment:
I am not sure this dependency is fair. These are just UART compatible chips, they are not architecture independent right? Especially the NS16650. Perhaps it would be better to just have them depend on `PLAT_KVM`?

Reply:
Actually that was not introduced in this PR, so I kept it as is to avoid changing too many things since this is very temporary. All this is wrapped in a huge `if (PLAT_KVM)` so in general we could remove it. On the other hand such architectural definitions are also set in all other drivers in this file, so there may be some reasoning there I'm not aware of. Besides all this will be reworked in plat re-arch, so maybe we should leave it as is for now. What do you think?

Reply:
> there may be some reasoning there I'm not aware of

Good point, we don't have much context here yet.

> should leave it as is for now

Yes, definetely. Too many last minute changes in the last few days anyway... :smile:.